pkgdb2/api/admin.py

```diff
@@ -151,7 +151,7 @@ def api_admin_actions():
             status=status,
             count=True,
         )
-    except pkgdblib.PkgdbException, err:  # pragma: no cover
+    except pkgdblib.PkgdbException as err:  # pragma: no cover
```

Comment:
I realize it is available in pkgdblib, but somehow it was failing in the tests despite that.
So, imho, either we are consistent and fix it everywhere or figure out more precisely why the tests were failing